image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -59,11 +59,7 @@ spec:
         - name: COLLECTOR_CONFIG
           value: '{"tlsConfig":{"caCertPath":"/var/run/secrets/stackrox.io/certs/ca.pem","clientCertPath":"/var/run/secrets/stackrox.io/certs/cert.pem","clientKeyPath":"/var/run/secrets/stackrox.io/certs/key.pem"}}'
         - name: COLLECTION_METHOD
-          {{- if (default ._rox.collector.forceCollectionMethod false) }}
-          value: {{ ._rox.collector.collectionMethod }}
-          {{- else }}
           value: CORE_BPF
```

Comment:
hmm good point, I think this is also a potential bug in 4.4 because we want to be able to configure NO_COLLECTION without forcing - forceCollection was only ever intended to override EBPF. I'll fix this.

Reply:
I see. Wondering if this change then would have any undesired impact for customers who were depending on the previous behavior. If I understand this correctly, this would be customers who were previously using
```
collectionMethod: "NO_COLLECTION"
forceCollectionMethod: "false"
```

Previously they would be using `CORE_BPF` in this situation and now they would be switching to `NO_COLLECTION`, right? I cannot reasonably comment on the implication of this -- but it might be worth a note for the CHANGELOG, given that we are changing customer-facing behaviour here. On the other hand, I don't know, is `NO_COLLECTION` used in the field?